src/main/java/com/minecolonies/coremod/entity/pathfinding/MinecoloniesAdvancedPathNavigate.java

```diff
@@ -115,7 +115,7 @@ public BlockPos getDestination()
     }
 
     @Nullable
-    public PathResult moveAwayFromXYZ(final BlockPos avoid, final double range, final double speedFactor)
+    public PathResult moveAwayFromXYZ(final BlockPos avoid, final double range, final double speedFactor, final boolean overrideDest)
```

Comment:
how about we call it `safeDestination` instead? And make it indicate whether the citizen should be able to reach the position(true), or doesnt know if it can reach it(false)